app/src/main/java/ca/llamabagel/transpo/search/data/SearchRepository.kt

```diff
@@ -22,6 +22,10 @@ import kotlinx.coroutines.withContext
 import javax.inject.Inject
 import javax.inject.Singleton
 
+private const val ROUTE_RESULT_LIMIT = 5L
+private const val STOP_RESULT_LIMIT = 5L
+private const val PLACE_RESULT_LIMIT = 5
```

Comment:
These should either all be `Int`s or all be `Long`s

Reply:
The Databases require Longs, Mapbox requires an Int

Reply:
Is it possible to force sqldelight to use Ints? `As Integer` maybe?